docs/content/latest/api/ysql/syntax_resources/the-sql-language/statements/savepoint_rollback.grammar.md

```diff
@@ -1,4 +1,4 @@
-```ebnf
```

Comment:
Curious why you're removing the language from these. Is it not working correctly?

Reply:
Hmm, when I regenerate the grammar files using `java -jar rrdiagram.jar content/latest/api/ysql/syntax_resources/ysql_grammar.ebnf   content/latest/api/ysql/syntax_resources/`, these '```ebnf'  lines get removed from the `savepoint_xxx.grammar.md` files. 
Should I just manually check out these files from this PR? or am i running the command incorrectly

Reply:
(I've fixed this in a separate PR, all good. No need for changes here.)